Containerfile

```diff
@@ -6,6 +6,6 @@ USER root
 
 RUN R -e "install.packages(c('astsa'), repos = 'https://cloud.r-project.org/', Ncpus = parallel::detectCores())"
 
-RUN mamba install -yc conda-forge "gluonts[torch,mxnet,pro,R]" orjson pyarrow
+RUN mamba install -yc conda-forge orjson pyarrow && pip install gluonts[ext,torch,mxnet,pro,R]
```

Comment:
I generally like pip to be on a separate RUN line from mamba so it's quicker/easier to debug if conda forge or PiPy is the problem when something breaks.